packages/create-plugin/src/constants.ts

```diff
@@ -19,6 +19,7 @@ export const FIXTURES_PATH = path.join(__dirname, '..', 'fixtures');
 export const TEMPLATE_PATHS: Record<string, string> = {
   app: path.join(TEMPLATES_DIR, 'app'),
   backend: path.join(TEMPLATES_DIR, 'backend'),
+  backendApp: path.join(TEMPLATES_DIR, 'backend-app'),
```

Comment:
nit: it would be nice to use the combination of app + backend instead. But I think we can refactor that in a follow up PR.

Reply:
Do you mean having all the backend-related stuff in one folder rather than two separate ones?